src/__tests__/utils/pgPool.ts

```diff
@@ -3,10 +3,12 @@ import { parse as parsePgConnectionString } from 'pg-connection-string';
 
 const pgUrl = process.env.TEST_PG_URL || 'postgres:///postgraphile_test';
 
-const pgPool = new Pool({
+export const poolConfig = {
   ...parsePgConnectionString(pgUrl),
   max: 15,
   idleTimeoutMillis: 500,
-});
+};
+
+const pgPool = new Pool(poolConfig);
```

Comment:
We need to swallow pool errors. https://github.com/graphile/postgraphile/blob/601a1086a2c10ebae4747364e199f2517d653861/src/postgraphile/postgraphile.ts#L264-L271

```suggestion
const pgPool = new Pool(poolConfig);
pgPool.on('error', () => {/* swallow error */});
```